packages/rhi-webgl/src/WebCanvas.ts

```diff
@@ -39,6 +40,10 @@ export class WebCanvas implements Canvas {
     }
   }
 
+  get pixelRatio(): number {
```

Comment:
Don't add this API，do not consider logic loopholes. If developer use width to modify it, the pixelRatio will not correct. Just add Input feature  in this PR.